e2e/testconfig/testconfig.go

```diff
@@ -150,3 +160,53 @@ func newDefaultSimappConfig(cc ChainConfig, name, chainID, denom string) ibc.Cha
 		NoHostMount:    false,
 	}
 }
+
+// defaultModifyGenesis will only modify governance params to ensure the voting period and minimum deposit
+// are functional for e2e testing purposes.
+func defaultModifyGenesis(denom string) func([]byte) ([]byte, error) {
+	return func(genbz []byte) ([]byte, error) {
+		genDoc, err := tmtypes.GenesisDocFromJSON(genbz)
+		if err != nil {
+			return nil, fmt.Errorf("failed to unmarshal genesis bytes into genesis doc: %w", err)
+		}
+
+		var appState genutiltypes.AppMap
+		if err := json.Unmarshal(genDoc.AppState, &appState); err != nil {
+			return nil, fmt.Errorf("failed to unmarshal genesis bytes into app state: %w", err)
+		}
+
+		cfg := simappparams.MakeTestEncodingConfig()
+		govv1beta1.RegisterInterfaces(cfg.InterfaceRegistry)
+		cdc := codec.NewProtoCodec(cfg.InterfaceRegistry)
+
+		govGenesisState := &govv1beta1.GenesisState{}
+		if err := cdc.UnmarshalJSON(appState[govtypes.ModuleName], govGenesisState); err != nil {
+			return nil, fmt.Errorf("failed to unmarshal genesis bytes into gov genesis state: %w", err)
+		}
+
+		// set correct minimum deposit using configured denom
+		govGenesisState.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(denom, govv1beta1.DefaultMinDepositTokens))
+
+		// set voting period to 10s
```

Comment:
looks like comment is not in sync with the code

Reply:
yea, originally I had it at 10s, but that wasn't sufficient for tests passing